.travis.yml

```diff
@@ -14,3 +14,7 @@ script:
 
 before_install:
   - "npm install -g npm@5.6.0"
+  - sudo apt-add-repository ppa:ondrej/php -y
+  - sudo apt-get update
+  - sudo apt-get install curl php7.0
```

Comment:
@bretterer - Can you take a look at this? I'm trying to install php 7 on travis but after running these commands, I still get the following error - 
`This package requires php ^7.0 but your PHP version (5.6.32) does not satisfy that requirement.`
Do I have to uninstall php5 before this?